website/docs/cdktf/concepts/constructs.mdx

```diff
@@ -93,17 +93,17 @@ app.synth();
 from constructs import Construct
 import cdktf
 import os
-import imports.kubernetes.provider as kubernetes
+from imports.kubernetes.provider import KubernetesProvider
```

Comment:
This code snippet is generated (see `<!-- #NEXT_CODE_BLOCK_SOURCE:python examples/python/documentation#constructs -->` above) – you should do the changes in [this file](https://github.com/hashicorp/terraform-cdk/blob/cf23f387160a066e579d5a87c19024387b4f8ff8/examples/python/documentation/main.py#L11) and run `yarn generate-docs:translations` in the repo root.